compiler/one-cmds/one-init

```diff
@@ -0,0 +1,262 @@
+                # And whitespaces btw key/value are unnecessary
+                # src ' key   =   value '
+                # dst ' key=value'
+                if self.delimiter in line:
+                    words = line.split(self.delimiter, maxsplit=1)
+                    # words[0]: ' key   '   -> ' key'
+                    # words[1]: '   value ' -> 'value'
+                    words[0] = words[0].rstrip()
+                    words[1] = words[1].strip()
+                    line = words[0] + self.delimiter + words[1]
+                cur_section.append(line)
+
+    def write(self, cfg_path):
+        # key: string, value: string
+        sections = {}
+
+        # convert first,
+        for section in self.sections.keys():
+            content = '\n'.join(self.sections[section])
+            sections[section] = content
+
+        with open(cfg_path, 'w') as f:
+            for k in sections.keys():
+                if k != self.default_key:
+                    f.write(k)
+                    f.write('\n')
+                f.write(sections[k])
+                f.write('\n')
+
+    # NOTE: section always start with '[' and end with ']'
+    def set(self, section, key, value):
+        # TODO: Handle exceptions for none of section and key
+        lines = self.sections[section]
+        for index, line in enumerate(lines):
+            # case: comments
+            if line.lstrip().startswith(self.comment_prefixes):
+                continue
+            # case: not strict config
+            if not self.delimiter in line:
+                continue
+
+            words = line.split(self.delimiter, maxsplit=1)
+            if key.strip() == words[0].strip():
+                lines[index] = words[0] + self.delimiter + value
+                return
+
+
+def _get_supported_map():
+    return {
+        'pb': 'tf',
+        'tflite': 'tflite',
+        'bcq': 'bcq',
+        'onnx': 'onnx',
+    }
+
+
+def _supported_model_ext(ext):
+    return ext in _get_supported_map()
+
+
+def _supported_model_type(model_type):
+    return model_type in _get_supported_map()
+
+
+# Return model type by file extension
+def _get_model_type(ext):
+    if not _supported_model_ext(ext):
+        raise NameError('Model extension (' + ext + ') doesn\'t support')
+    return _get_supported_map()[ext]
+
+
+# NOTE: Do not use '_add_default_arg'. '-C' and '-S' options are unnecessary.
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to create a new config file')
+
+    # version
+    parser.add_argument(
+        '-v',
+        '--version',
+        action='store_true',
+        help='show program\'s version number and exit')
+
+    # model path
+    parser.add_argument(
+        '--model_path',
+        required=False,
+        type=str, default='',
+        help='the model path to be used for creating config file')
+
+    return parser
+
+
+def _parse_arg(parser):
+    # NOTE: Trick for parsing args
```

Comment:
This seems not good but I cannot avoid it. To support cases where no argument and any argument with argparser, not sys.argv, I fix it like this. If you have an any better option, I'll change it.